pkg/prometheus/config/config.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // Configuration describes the options to customize the storage behavior.
 type Configuration struct {
-	ServerURL                string
+	ServerURL                string `mapstructure:"endpoint"`
```

Comment:
``mapstructure:"endpoint"``

@yurishkuro Do we need this ? When I am trying to pass the ip address in docker-compose it tells implicit type-casting will fail  https://github.com/open-telemetry/opentelemetry-collector/issues/9532

Reply:
we need this for all fields, but to make the test work we at minimum need the endpoint, because without it the default is localhost:9090 which points back to Jaeger itself, not to Prometheus server. I am about to submit a few fixes, but I still haven't gotten the test to worl

Reply:
So the tags are what we use to retrieve the value from the config file ?

Reply:
tags are the names we can use in the config file, if we don't have them we have to refer by field name like `ServerURL`, which is not consistent with the rest of configs.

Reply:
Understood.